src/Payments/Payment.php

```diff
@@ -0,0 +1,487 @@
+     * @var \DateTime
+     * @JMS\Type("DateTime<'Y-m-d'>")
+     * @JMS\SerializedName("date")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $date;
+
+    /**
+     * @var int
+     * @JMS\Type("int")
+     * @JMS\SerializedName("initial")
+     * @JMS\Groups({"response"})
+     */
+    private $initial;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("type")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"response"})
+     */
+    private $type;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("payment_method")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $paymentMethod;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("payment_type")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $paymentType;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("account")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $account;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("value_pln")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $valuePln;
+
+    /**
+     * @var float
+     * @JMS\Type("double")
+     * @JMS\SerializedName("currency_exchange")
+     * @JMS\Groups({"response"})
+     */
+    private $currencyExchange;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     * @JMS\SerializedName("currency_label")
+     * @JMS\Groups({"response"})
+     */
+    private $currencyLabel;
+
+    /**
+     * @var \DateTime
+     * @JMS\Type("DateTime<'Y-m-d'>")
+     * @JMS\SerializedName("currency_date")
+     * @JMS\Groups({"response"})
+     */
+    private $currency_date;
+
+    /**
+     * @var int
+     * @JMS\Type("integer")
+     * @JMS\SerializedName("notes")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"request", "response"})
+     */
+    private $notes;
+
+    /**
+     * @var TagIds
+     * @JMS\Type("Webit\WFirmaSDK\Tags\TagIds")
+     * @JMS\SerializedName("tags")
+     * @JMS\Groups({"request", "response"})
+     */
+    private $tags;
+
+    /**
+     * @var InvoiceId
+     * @JMS\Type("Webit\WFirmaSDK\Invoices\InvoiceId")
+     * @JMS\SerializedName("invoice")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"response"})
+     */
+    private $invoice;
+
+    /**
+     * @var ExpenseId
+     * @JMS\Type("Webit\WFirmaSDK\Expenses\ExpenseId")
+     * @JMS\SerializedName("expense")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"response"})
+     */
+    private $expense;
+
+    /**
+     * @var SeriesId
+     * @JMS\Type("Webit\WFirmaSDK\Series\SeriesId")
+     * @JMS\SerializedName("series")
+     * @JMS\XmlElement(cdata=false)
+     * @JMS\Groups({"response"})
+     */
+    private $series;
```

Comment:
Zmień proszę nazwę na `seriesId` (skoro to ID).

Reply:
Zmienione.